vyxal/helpers.py

```diff
@@ -42,17 +42,6 @@ def case_of(value: str) -> int:
     return -1
 
 
-@lazylist
-def chop(it: VyIterable, n: int) -> LazyList:
```

Comment:
Why remove from helpers? It's the definition of a helper.

Reply:
Because at one point it needed something from elements. It doesn't now.